code/game/machinery/computer/robot.dm

```diff
@@ -96,6 +102,18 @@
 						to_chat(usr, span_danger("Cyborg locked by an user with superior permissions."))
 			else
 				to_chat(usr, span_danger("Access Denied."))
+
+		if("killbot") //Malf AIs, and AIs with a combat upgrade, can detonate their cyborgs remotely.
+			if(!isAI(usr))
+				return
+			var/mob/living/silicon/ai/arty = usr
+			if(!arty.malf_picker)
+				return
+			var/mob/living/silicon/robot/rob = locate(params["ref"]) in GLOB.silicon_mobs
```

Comment:
You need to `istype` check this afterwards, otherwise an invalid ref will return null which will just runtime later on.